src/tools/CMakeLists.txt

```diff
@@ -29,6 +30,7 @@ install(TARGETS ceph-osdomap-tool DESTINATION bin)
 add_executable(ceph-monstore-tool ceph_monstore_tool.cc)
 target_link_libraries(ceph-monstore-tool os global Boost::program_options)
 install(TARGETS ceph-monstore-tool DESTINATION bin)
+if(WITH_TESTS)
```

Comment:
please just move https://github.com/ceph/ceph/blob/master/src/tools/CMakeLists.txt#L25-L33 out of the `WITH_TEST` block.

Reply:
Done!